source/braille.py

```diff
@@ -27,73 +27,181 @@
+	# tool tip.
+	controlTypes.ROLE_TOOLTIP: _("tltip"),
 	# Translators: Displayed in braille for an object which is a
 	# link.
 	controlTypes.ROLE_LINK: _("lnk"),
 	# Translators: Displayed in braille for an object which is a
-	# dialog.
-	controlTypes.ROLE_DIALOG: _("dlg"),
-	# Translators: Displayed in braille for an object which is a
 	# tree view.
 	controlTypes.ROLE_TREEVIEW: _("tv"),
 	# Translators: Displayed in braille for an object which is a
+	# tree view item.
+	controlTypes.ROLE_TREEVIEWITEM: _("tvitem"),
+	# Translators: Displayed in braille for an object which is a
+	# tab control.
+	controlTypes.ROLE_TABCONTROL: _("tabctl"),
+	# Translators: Displayed in braille for an object which is a
+	# progress bar.
+	controlTypes.ROLE_PROGRESSBAR: _("prgbar"),
+	# Translators: Displayed in braille for an object which is a
+	# scroll bar.
+	controlTypes.ROLE_SCROLLBAR: _("scrlbar"),
+	# Translators: Displayed in braille for an object which is a
+	# status bar.
+	controlTypes.ROLE_STATUSBAR: _("stbar"),
+	# Translators: Displayed in braille for an object which is a
 	# table.
-	controlTypes.ROLE_TABLE: _("tb"),
+	controlTypes.ROLE_TABLE: _("tbl"),
+	# Translators: Displayed in braille for an object which is a
+	# tool bar.
+	controlTypes.ROLE_TOOLBAR: _("tlbar"),
 	# Translators: Displayed in braille for an object which is a
+	# drop down button.
+	controlTypes.ROLE_DROPDOWNBUTTON: _("ddbtn"),
+	# Displayed in braille for an object which is a
 	# separator.
-	controlTypes.ROLE_SEPARATOR: _("-----"),
+	controlTypes.ROLE_SEPARATOR: u"⠤⠤⠤⠤⠤",
 	# Translators: Displayed in braille for an object which is a
-	# graphic.
-	controlTypes.ROLE_GRAPHIC: _("gra"),
+	# block quote.
+	controlTypes.ROLE_BLOCKQUOTE: _("blq"),
+	# Translators: Displayed in braille for an object which is a
+	# document.
+	controlTypes.ROLE_DOCUMENT: _("doc"),
+	# Translators: Displayed in braille for an object which is a
+	# application.
+	controlTypes.ROLE_APPLICATION: _("app"),
+	# Translators: Displayed in braille for an object which is a
+	# grouping.
+	controlTypes.ROLE_GROUPING: _("grp"),
+	# Translators: Displayed in braille for an object which is a
+	# embedded object.
+	controlTypes.ROLE_EMBEDDEDOBJECT: _("embedded"),
+	# Translators: Displayed in braille for an object which is a
+	# end note.
+	controlTypes.ROLE_ENDNOTE: _("enote"),
+	# Translators: Displayed in braille for an object which is a
+	# foot note.
+	controlTypes.ROLE_FOOTNOTE: _("fnote"),
+	# Translators: Displayed in braille for an object which is a
+	# terminal.
+	controlTypes.ROLE_TERMINAL: _("term"),
+	# Translators: Displayed in braille for an object which is a
+	# section.
+	controlTypes.ROLE_SECTION: _("sect"),
+	# Translators: Displayed in braille for an object which is a
+	# toggle button.
+	controlTypes.ROLE_TOGGLEBUTTON: _("tbtn"),
+	# Translators: Displayed in braille for an object which is a
+	# split button.
+	controlTypes.ROLE_SPLITBUTTON: _("sptbtn"),

```

Comment:
Probably. I thought that's actually what we used, but we must have had an even brighter idea at some point. :)
Keeping in line with drbtn and tgbtn, would spbtn be an option? Probably not, because it could be confused with spin button. So splbtn then?